experimental/packages/opentelemetry-sdk-metrics-base/src/ObservableResult.ts

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as metrics from '@opentelemetry/api-metrics-wip';
+import { AttributeHashMap } from './state/HashMap';
+
+export class ObservableResult implements metrics.ObservableResult {
+  buffer = new AttributeHashMap<number>();
+
+  observe(value: number, attributes: metrics.Attributes = {}): void {
```

Comment:
Should we document that if more than one measurement with same attributes is observed in one collection cycle then last one is kept?

Reply:
Submitted: https://github.com/open-telemetry/opentelemetry-js/pull/2712